app/src/main/AndroidManifest.xml

```diff
@@ -10,7 +10,8 @@
         android:label="@string/app_name"
         android:supportsRtl="true"
         android:theme="@style/Theme.NomicGameManagerMobile"
-        tools:targetApi="31">
+        tools:targetApi="31"
+        android:usesCleartextTraffic="true">
```

Comment:
I added that since the local development with the api is done of http and there doesn't seem to be a super clear way of implementing a way for just a debug mode.  I did add it to the loss threads jira page though